Ambit/Source/AmbitUtils/JsonHelpers.cpp

```diff
@@ -14,18 +14,18 @@
 
 #include "JsonHelpers.h"
 
-#include <stdexcept>
-
 #include "MenuHelpers.h"
 #include "Serialization/JsonSerializer.h"
 
+#include <stdexcept>
+
 namespace FJsonHelpers
 {
-    FString SerializeJson(TSharedPtr<FJsonObject> JsonObject)
+    FString SerializeJson(const TSharedPtr<FJsonObject>& JsonObject)
```

Comment:
blarg. this has implications and would recommend NOT automatically making this change. This is saying that this function only needs to "use" the JsonObject that some other thing owns, but in actuality we DO want SerializeJson to also be considered an "owner" and reference count to be incremented so that if the owning class goes out of scope this doesn't die.